assets/stylesheets/comments.scss

```diff
@@ -47,6 +47,16 @@
         }
     }
 
+    .openqa-bugref {
+        position: relative;
+        background-color: #eee;
+        color: #666666;
```

Comment:
this looks like some duplication. We already had colors in before. So why should we need to define those settings here again?

Reply:
We don't currently have styling for bugrefs

Reply:
I don't see how this was resolved. Can you clarify?

Reply:
@kalikiana's comment explained it.